EGC-zambrano/decide-part-zambrano
# Feature/census csv

With this pull request, the administrator is finally able to reduce the time-consuming task that is adding voters to a census by importing it via a CSV file.
As always, run the commands:
```
pip install -r requirements.txt
cd decide
python manage.py makemigrations
python manage.py migrate
```
To test this functionality do the following:
1. Create a CSV file in the repository of your choice:
```
nano census.csv
```
2. Each line has to have the format of votingId,voterId. For example:
```
1,1
1,3
5,6
```
3. Go to the admin panel, navigate to Census, and click Import Census via CSV.
4. There, import census.csv.
5. Check the that the census instances have been created correctly.

Finally, run the tests with 
```
python manage.py test census
```
